source/administration/changelog.md

```diff
@@ -4,6 +4,171 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Idiomatic error handling
+
+#### User Interface
+ - Added ability to remove profile pictures in Account Settings.
+ - Added a new bar that shows progress on uploads.
+ - Added a new badge to the profile popover that indicates a user is a server admin.
+ - Added different experimental channel sidebar reorganization settings, such as the ability to sort channels by recent posts.
+ - Added an admin badge to profile popover to allow users to see if a user is a server admin.
+ - Added an option to be able to clear search results.
+
+#### Notifications
+ - Enabled push notifications to use TPNS by default.
+ - Added a channel notification setting to disable @-channel @-here @-all notifications in specific channels.
+
+#### Performance
+ - Return fewer user autocomplete matches.
+
+#### Plugins
+ - Added "min_server_version" in plugin.json manifest to enable built-in control for preventing load/enable of plugin if server version is not met.
+ - Added ability for plugins to add channel header tooltips.
+ - Stop hashing plugin keys on write. **////XXXX**
+
+#### Bulk Import/Export
+ - Added capability to export reactions of posts during bulk export.
+ - Added bulk import email intervals to specify an email batching interval during bulk import.
+ - Included favorite channels in bulk export.
+ - Added User Notify Props and per-channel NotifyProps in Bulk Export.
+ - Added capability to bulk export custom emojis.
+
+#### Command Line Interface (CLI)
+ - Added ``command create`` CLI command to check the ``EnableOnlyAdminIntegrations`` setting and to only allow team admins to create slash commands if this is set to ``true``.
+ - Added ``team search`` CLI command to search for teams based on team names.
+ - Added ``webhook list`` CLI command to allow listing webhooks for a team or multiple teams.
+ - Added ``team list`` CLI command for listing commands under a team.
+ - Added ``command delete`` CLI command to allow deleting slash commands.
+ - Added ``team archive`` CLI command to archive a team by name.
+ - Added ``create webhook-incoming`` CLI command to create incoming webhooks.
+ - Added ``command create`` CLI command to create slash commands.
+ - Added ``modify-incoming webhook`` CLI command to modify incoming webhooks.
+ - Added ``webhook create-outgoing`` CLI command to create outgoing webhooks.
+ - Added ``webhook delete`` CLI command to delete webhooks.
+
+#### Slash Commands
+ - Added support for multiple responses to the User Interface from a slash command.
+ - Added an option to send a message when an invalid slash command is entered.
+
+#### Administration
+ - Terms of Service feature improvements.
+ - Removed "System Console > Plugins > Configuration" page and moved enabling plugins setting to the Management page.
+
+#### Enterprise Edition (E20)
+ - Data Retention was promoted out of beta.
+
+### Bug Fixes
+
+ - Fixed an issue where pinned post list refreshed when a user posted a new message.
```

Comment:
Hm, did we only fix one bug in v5.6? Were the others regressions and not affecting v5.5?